transform/gtex/samples.py

```diff
@@ -0,0 +1,69 @@
+from bmeg.util.cli import default_argument_parser
+from bmeg.emitter import JSONEmitter
+from bmeg.ioutils import read_tsv
+from bmeg.vertex import Aliquot, Biosample, Project, Individual
+from bmeg.edge import AliquotFor, BiosampleFor, InProject
+
+
+parser = default_argument_parser()
```

Comment:
Looks like this parser block could be removed.

Reply:
Fixed